### DISCUSSION TITLE
Autoincrement fails in one database but works in the other

### DISCUSSION THREAD

**Discussion by wvhulle** [prisma/prisma#16907] [Q&A] [answered]
When I try to create a row in my measurement table with `await prisma.measurement.create({data: {}})`, I get 
```
Invalid `prisma.measurement.create()` invocation:
Unique constraint failed on the fields: (`id`)
```
My table looks like
```
model Measurement {
    id            Int       @id @default(autoincrement())
    sampleLabel   String?
    sample        Sample?   @relation("measurementToSample", fields: [sampleLabel], references: [label], onDelete: Cascade)
    variableLabel String?
    variable      Variable? @relation("measurementToVariable", fields: [variableLabel], references: [label], onDelete: Cascade)
    value         String?
}
```

It also fails for createMany. However, I have copied the exact table and rows to a new project and database and there the problem doesn't happen. It seems like the id is not autoincremented in one database but not in the other. How is that possible?

**Comment by nurul3101** [comment]
Hello @wvhulle 👋 
Do both projects use the same Prisma version and the same database version?

**Comment by wvhulle** [comment, marked as answer]
I figured it out. Somehow the serial column was not correct anymore. I was
using the same versions of prisms and postgres

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
